lib/colorls/core.rb

```diff
@@ -1,7 +1,7 @@
 module ColorLS
   class Core
     def initialize(input=nil, all: false, report: false, sort: false, show: false,
-      one_per_line: false, long: false, almost_all: false, tree: false, help: false, colors: [])
+      one_per_line: false, long: false, almost_all: false, tree: false, colors: [])
```

Comment:
"better" is just a matter of taste, I guess.

But usually you want to avoid any tedious or repetitious (maintenance) work; so I wanted to keep this DRY.

Also I did not understand the message of the helplog, what the "command" part was referring to... something about grep and the README? :confused:

BTW, here's the output after applying this change:

```
Usage:  colorls [OPTION]... [FILE]...

    -a, --all                        do not ignore entries starting with .
    -A, --almost-all                 do not list . and ..
    -l, --long                       use a long listing format
    -t, --tree                       shows tree view of the directory
    -r, --report                     show brief report
    -1                               list one file per line
    -d, --dirs                       show only directories
    -f, --files                      show only files

sorting options:

    -sMODE                           sort (f)iles or (d)irectories first
        --sort-dirs, --group-directories-first
                                     sort directories first
        --sort-files                 sort files first

general options:

    -h, --help                       Prints this help
        --version                    Show version
```